milestone.md

```diff
@@ -0,0 +1,199 @@
+Kamień milowy do: 6.12.2022r
+
+# Założenia
+- Drużyny jednoosobowe, czyli każdy gra przeciw każdemu
+
+## Stany gry
+Gra może znajdować się w jednym z następujących stanów:
+- przygotowanie (domyślny przy tworzeniu)
+- oczekiwanie na graczy
+- rozgrywka
+- zakończona
+
+Na podstawie stanu decydowane jest jakie akcje w endpointach mogą być wykonywane, a jakie nie. 
+
+Wpływ stanów na uprawnienia:
+- przygotowanie
+
+  GM tworzy i edytuje zadania
+
+  Gracz nie widzi gry na liście
+
+- oczekiwanie na graczy
+
+  Gracz widzi grę i jej opis
+
+  Gracz nie widzi zadań
+
+  GM nie może edytować zadań
+
+- rozgrywka
+
+  Gracz widzi grę i jej opis (możliwość dołączania do gry w trakcie jej trwania)
+
+  Gracz widzi zadania i może przesyłać odpowiedzi
+
+  GM ocenia odpowiedzi do zadań
+
+  GM nie może edytować zadań
+
+- zakończona
+
+  GM widzi grę na liście
+
+  Gracz nie widzi gry na liście
+
+  Nie można nic edytować
+
+## Zakończenie gry
+Możliwy jest wybór sposobu zakończenia gry:
+- manualny
+    
+    GM według uznania kończy grę. Niezależnie czy został wybrany którykolwiek z pozostałych trybów, możliwe jest manualne zakończenie gry.
+
+- czasowy
+
+    Gra kończy się o ustalonej dacie i godzinie.
+
+- zdobycia liczby punktów
+
+    Ustalany jest próg punktowy. Jeżeli którakolwiek drużyna zdobędzie taką lub większą liczbę punktów, gra kończy się.
+
+- wyczerpanie zadań
+
+    Jeżeli istnieje drużyna, która nie ma dostępnych zadań bez zgłoszonej zatwierdzonej odpowiedzi, gra kończy się.
+    
+
+
+## Konstrukcja zadań
+Zadanie składa się z:
```

Comment:
To nie jest wypunktowane - skleja się.

Reply:
A czego mu właściwie brakuje, żeby nie sklejało?

Reply:
No idea, poeksperymentuj.